packages/grafana-data/src/types/icon.ts

```diff
@@ -6,7 +6,7 @@ export const availableIconsIndex = {
   okta: true,
   discord: true,
   hipchat: true,
-  'google-hangouts-alt': true,
+  'google-chat-alt': true,
```

Comment:
This icon doesn't exist, we can either keep the existing hangouts icon or use a more generic `google` icon.

https://iconscout.com/unicons/explore/line